controllers/volumereplicationgroup_controller.go

```diff
@@ -41,6 +43,12 @@ import (
 	"github.com/ramendr/ramen/controllers/volsync"
 )
 
+const (
+	volumeReplicationClass = "VolumeReplicationClass"
+	volumeReplication      = "VolumeReplication"
```

Comment:
Just for more clear code. Why are you asking about these constants and not about crdCheckInterval?

Reply:
Separating the configuration from the code using it makes it easier to change the configuration. Adding constants for the CRD name does not help anyone. It could be 
helpful if the constant was requiredCRDS, so we can add another CRD later without
changing the code.

Reply:
Done